build/Helix/ConvertWttLogToXUnit.cs

```diff
@@ -0,0 +1,287 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+
+namespace HelixTestHelpers
+{
+    public static class TestResultParser
+    {
+        public class TestPass
+        {
+            public TimeSpan TestPassExecutionTime { get; set; }
+            public List<TestResult> TestResults { get; set; }
+        }
+
+        public class TestResult
+        {
+            public string Name { get; set; }
+            public bool Passed { get; set; }
+            public bool CleanupPassed { get; set; }
+            public TimeSpan ExecutionTime { get; set; }
+            public string Details { get; set; }
+        }
+
+        public static void ConvertWttLogToXUnitLog(string wttInputPath, string xunitOutputPath)
+        {
+            var testPass = TestResultParser.ParseTestWttFile(wttInputPath, true);
+            var results = testPass.TestResults;
+
+            int resultCount = results.Count;
+            int passedCount = results.Where(r => r.Passed).Count();
+            int failedCount = resultCount - passedCount;
+
+            var root = new XElement("assemblies");
+
+            var assembly = new XElement("assembly");
+            assembly.SetAttributeValue("name", "MUXControls.Test.dll");
+            assembly.SetAttributeValue("test-framework", "TAEF");
+            assembly.SetAttributeValue("run-date", DateTime.Now.ToString("yyyy-mm-dd"));
+            assembly.SetAttributeValue("run-time", DateTime.Now.ToString("hh:mm:ss"));
```

Comment:
It may confuse the user, I guess the correct time may be Now-testPass.TestPassExecutionTime.TotalSeconds

Reply:
I'll change.